frontend/src/Pages/Buyer/HomePageSearch.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect, useState } from 'react'
+
+interface HomePageSearch {
+    name: { name: string }[]
+}
+
+const data: HomePageSearch = {
```

Comment:
Component looks great, @AAlchi ! 

Dark Mode makes a lot of lighter content dark -- 
![image](https://github.com/miguelaenlle/merch-order-request-form/assets/66172204/42314f6b-209b-481b-beef-333ba62075d0)

This can be fixed in a separate task at a later time.